.travis.yml

```diff
@@ -2,7 +2,6 @@
 language: ruby
 
 rvm:
-  - 1.8.7
```

Comment:
Re-added and pushed

On Fri, Mar 11, 2016 at 4:28 PM Garrett Honeycutt <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/ghoneycutt/puppet-module-hosts/pull/42#discussion_r55891114>
> :
>
> > @@ -2,7 +2,6 @@
> >  language: ruby
> >
> >  rvm:
> > -  - 1.8.7
>
> Go ahead and leave this in so we can see the errors and I'll work to keep
> this supported.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ghoneycutt/puppet-module-hosts/pull/42/files#r55891114>
> .
>